lxc/list.go

```diff
@@ -367,8 +367,19 @@ func (c *listCmd) statusColumnData(cInfo shared.ContainerInfo, cState *shared.Co
 	return strings.ToUpper(cInfo.Status)
 }
 
+func cIsActive(cInfo shared.ContainerInfo) bool {
```

Comment:
Quoting Stéphane Graber (notifications@github.com):
> > @@ -367,8 +367,19 @@ func (c *listCmd) statusColumnData(cInfo shared.ContainerInfo, cState *shared.Co
> >  	return strings.ToUpper(cInfo.Status)
> >  }
> >  
> > +func cIsActive(cInfo shared.ContainerInfo) bool {
> 
> Should be func (c *listCmd) isActive(...)
> 
> We try not to keep all functions tied to structs unless they are shared with other files.

That's odd - the fn doesn't use the listcmd at all.